server/middleware/accountsMidware.js

```diff
@@ -1,7 +1,20 @@
 import accounts from '../models/accounts';
-
-const Account = {
-  isValidType(req, res, next) {
+/**
+ * Checks for errors before performing any action on an account
+ * @class Account
+ */
+class Account {
+  /**
+   *
+   *
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns
+   * @memberof Account
```

Comment:
If this is a validation class I do not think that `Account` is the best name for it